mmpose/datasets/pipelines/shared_transform.py

```diff
@@ -142,3 +150,164 @@ def __repr__(self):
         """Compute the string representation."""
         return (f'{self.__class__.__name__}('
                 f'keys={self.keys}, meta_keys={self.meta_keys})')
+
+
+@PIPELINES.register_module()
+class Normalize(object):
```

Comment:
Is Normalize used anywhere?

Reply:
Not used for now. 
It looks like a standard pipeline, so I also added it.

Reply:
Let's add it later.

Currently all normalization comes after the ToTensor, using NormalizeTensor.
Either we keep this convention, or we mv all normalization before ToTensor to use Normalize.

Reply:
OK.